docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+**bash**
+
+In `bash`, put a dollar sign before the argument to allow single quotes. Escape special characters with a backslash and use a double backslash for semicolons to escape them from the TestCafe argument parser:
+
+```sh
+testcafe $'chrome:emulation:userAgent=\'Mozilla/5.0 (Windows NT 10.0\\; Win64\\; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36\'' test.js
+```
+
+**PowerShell**
+
+In `PowerShell`, escape special characters with a single quote and use a backslash for semicolons to escape them from the TestCafe argument parser:
+
+```sh
+testcafe 'chrome:emulation:userAgent=''Mozilla/5.0 (Windows NT 10.0\; Win64\; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36''' test.js
+```
+
+### Emulator Parameters
+
+Use the following parameters to configure Chrome device emulator. Specify them after the `emulation` parameter as in the examples above.
+
+Parameter                      | Type   | Description             | Default
+------------------------------ | ------ | ----------------------- | -------
+`device` *(optional)*       | String  | The emulated device name (see the full list of supported devices in DevTools -> ⠇-> Settings -> Devices). | No default value.
+`width` *(optional)*        | Number  | The device screen width in pixels. | The chosen device's width. If the `device` parameter is not set, the default browser's width.
+`height` *(optional)*       | Number  | The device screen height in pixels. | The chosen device's height. If the `device` parameter is not set, the default browser's height.
+`scaleFactor` *(optional)*  | Number  | Device scale factor value. | Depends on a chosen `device` or your system parameters.
+`mobile` *(optional)*       | Boolean | Whether to emulate a mobile device. This includes the viewport meta tag, overlay scrollbars, text autosizing and more. | `true` if a mobile device is set via the `device` property. Otherwise `false`.
+`orientation` *(optional)*  | `vertical` &#124; `horizontal` | The device orientation | `vertical`
+`userAgent` *(optional)*    | String  | The user agent string | The user agent string of the selected `device` or the browser.
+`touch` *(optional)*        | Boolean | Enables or disables the touch event emulation. | `true` if a touch-supported device is set via the `device` property or your system supports touch events. Otherwise `false`.
+`cdpPort` *(optional)*      | Number  | A port (0-65535) used for the Chrome Debugging Protocol. | `9222` if you load a user profile with the [:userProfile](#user-profiles) or `--user-data-dir` flag. Otherwise, TestCafe automatically assigns a free port.
+
+## User Profiles
+
+By default, TestCafe launches browsers (Google Chrome and Mozilla Firefox so far) with a clean profile, i.e. without extensions, bookmarks and other profile settings. This was done to minimize the influence of profile parameters on test runs.
```

Comment:
Can we say:
By default, TestCafe launches Google Chrome and Mozilla Firefox with a clean profile (i.e. without extensions, bookmarks and other profile settings) - to minimize profile parameter influence on test runs.